src/components/Guilds/Sidebar/StakeTokens.tsx

```diff
@@ -87,14 +93,46 @@ const ButtonLock = styled(Button)`
 
 export const StakeTokens = ({ onJoin }) => {
   const [dxdValue, setDXDValue] = useState(0);
+  const {
+    data: { timeForExecution, proposalTime, name, token },
+    error,
+  } = useConfig();
+
   return (
     <GuestContainer>
       <DaoBrand>
         <DaoIcon src={dxIcon} alt={'DXdao Logo'} />
-        <DaoTitle>DXdao</DaoTitle>
+        <Loading text loading={!name && !error}>
+          <DaoTitle>{error ? 'DxDao' : name}</DaoTitle>

```

Comment:
Dont think we need a backup placeolder when it fails here, eventually we'll have a proper error screen also

Reply:
Sure, will remove this default string.